### ISSUE TITLE
Uses for spare Pi..

### ISSUE THREAD

**Issue by CCOSTAN** [CCOSTAN/Home-AssistantConfig#365]
Open to ideas ..

http://blissflixx.rocks/ - Use as a streamer ? Maybe something outside?

**Comment by d0ugal** [comment]
I have been experimenting with https://volumio.org and found it great. The sound output from the pi is terrible tho’ - so you may want to get a DAC...(I was getting lots of crackling when nothing was playing). I ordered a JustBoom DAC, so I’ll know soon if that helps.

**Comment by JamesMcCarthy79** [comment]
@d0ugal agreed on the sound performance from the Pi, I use a a HiFiBerry Amp2+ makes a massive improvement, especially if outdoor/entertainment is the intended use case.

**Comment by CCOSTAN** [comment]
@JamesMcCarthy79 I'm basically just trying to figure out what do with with my Pis. :) I have a Pi 3 and 2 Pi zeros..

**Comment by CCOSTAN** [comment]
The wrinkle is it needs to be something that I just can't run as a docker image on the main system.   Magic Mirror maybe?  Maybe this #311

**Comment by JamesMcCarthy79** [comment]
I'm about to use one of my spare Pi Zero with a cheap ultrasonic sensor to measure the water level in my coffee machine :).

**Comment by JamesMcCarthy79** [comment]
Also turn your fridge into a smart fridge 

https://www.hackster.io/pamruta/smart-fridge-1baf56

**Comment by CCOSTAN** [comment]
Closing this original brainstorming issue now that the larger spare Raspberry Pi has an active infrastructure role. Current evaluation of the two remaining Pi Zeros will continue in #1947, using explicit no-redundancy, maintainability, and acceptance criteria. The older #376 concept remains available as historical input.
